app/src/main/res/layout/fragment_suggested_edits_image_recommendation_item.xml

```diff
@@ -319,25 +318,36 @@
         android:elevation="14dp"
         tools:visibility="gone">
 
+        <androidx.appcompat.widget.AppCompatImageView
```

Comment:
fyi, there's no need to use the fully-qualified `AppCompatImageView`. Just the plain `ImageView` will do the same thing.